README.md

```diff
@@ -1,161 +1,120 @@
+example-infinispan-54c66fd755-7c4zc    0/1       ContainerCreating   0          4s
+example-infinispan-54c66fd755-8gbxf    0/1       ContainerCreating   0          5s
+infinispan-operator-69d7d4469d-f62ws   1/1       Running             0          3m
+example-infinispan-54c66fd755-8gbxf    1/1       Running             0          8s
+example-infinispan-54c66fd755-7c4zc    1/1       Running             0          8s
+example-infinispan-54c66fd755-28lvx    1/1       Running             0          8s
 ```
 
-### Custom Resource Definition
+Now it's time to have some fun. Let's see the Infinispan operator in action.
 
-The Infinispan resource definition is detailed at [infinispan-types.go](https://github.com/infinispan/infinispan-operator/blob/master/pkg/apis/infinispan/v1/infinispan_types.go).
+Change the cluster size in `deploy/cr/cr_minimal.yaml` and then apply it again. The Infinispan operator scales the number of nodes in the cluster up or down.
 
-Below are some example of creating different kinds of clusters:
+### Custom Resource Definitions
 
-#### Minimal custer
+Custom resources allow you to dynamically configure Infinispan clusters using the Kubernetes API.
 
-A cluster created using ```cloud.xml``` configuration can be created with:
+Infinispan resources are defined in [infinispan-types.go](https://github.com/infinispan/infinispan-operator/blob/master/pkg/apis/infinispan/v1/infinispan_types.go).
+
+#### Minimal Configuration
+Use this resource to create Infinispan clusters using the `cloud.xml` configuration. This is the default configuration and uses the Kubernetes JGroups stack to form clusters with the `KUBE_PING` protocol.
 
 ```yaml
 apiVersion: infinispan.org/v1
 kind: Infinispan
 metadata:
-  name: example-minimal
+  name: example-infinispan-minimal
 spec:
   size: 3
 ```
 
-#### Using an internal configuration
-
-To point to any pre-existing configuration from Infinispan, for e.g. ```clustered.xml```:
+#### Infinispan Configuration
+Use this resource to create Infinispan clusters using a configuration such as `clustered.xml` from `/opt/jboss/infinispan-server/standalone/configuration/`.
 
 ```yaml
 apiVersion: infinispan.org/v1
 kind: Infinispan
 metadata:
-  name: example-minimal
+  name: example-infinispan-config
 config:
   name: clustered.xml
 spec:
   size: 3
 ```
 
-
-#### Using a config map
-
-
-To use a file ```my-config.xml``` stored in a config map with name ```my-config-map``` as configuration:
+#### Custom Configuration
+Use this resource to create Infinispan clusters with custom configuration via a ConfigMap. For example, configure clusters with `my-config.xml` with a ConfigMap named `my-config-map`.
 
 ```yaml
 apiVersion: infinispan.org/v1
 kind: Infinispan
 metadata:
-  name: example-infinispan
+  name: example-infinispan-custom
 config:
   sourceType: ConfigMap
   sourceRef:  my-config-map
   name: my-config.xml
 spec:
   size: 3
-
-```
-
-### Running tests
```

Comment:
Why remove this section and the one about making releases? It's useful for contributors and team members.